sdbuild/packages/xrt/qemu.sh

```diff
@@ -0,0 +1,36 @@
+# Set up some environment variables as /etc/environment
+# isn't sourced in chroot
+
+set -e
+set -x
+
+# to be added into the main flow
+apt-get update
+apt-get -y install libgtest-dev libboost-all-dev uuid-dev libyaml-dev 
+apt-get -y install opencl-headers ocl-icd-dev  ocl-icd-opencl-dev
```

Comment:
@PeterOgden  For line 8 ~ 10, should we add them into the multistrap? Let me know so I can fix this.

Reply:
Seems reasonable to me